### DISCUSSION TITLE
FDS version in the log file

### DISCUSSION THREAD

**Discussion by ahsadeghi** [firemodels/fds#12754] [Q&A]
Hello,
I have compiled the latest FDS today, and the following version is written in the log file

 Current Date     : April  5, 2024  17:43:13
 Revision         : FDS6.7.3-9780-gfccda53-master
 Revision Date    : Thu Apr 4 19:08:52 2024 -0400
 Compiler         : GCC version 11.4.0
 Compilation Date : Apr 05, 2024  17:37:33

but it should be FDS 6.9. Do you know why this happens? should I ignore it?

Regards,
Hosein

**Comment by drjfloyd** [comment]
you may need to update tags in your local git repository.  Do a
git tag

at a command line when in your local fds repository.  If you don't see the tag for 6.9 then do a

git fetch --tags
